pyccl/halos/profiles_cib.py

```diff
@@ -77,8 +77,9 @@ class HaloProfileCIBShang12(HaloProfile):
         L0 (float): luminosity scale (in
             :math:`{\\rm Jy}\\,{\\rm Mpc}^2\\,M_\\odot^{-1}`).
     """
-    __repr_attrs__ = ("cM", "nu", "alpha", "T0", "beta", "gamma", "s_z",
-                      "l10meff", "sigLM", "Mmin", "L0", "precision_fftlog",)
+    __repr_attrs__ = __eq_attrs__ = (
+        "cM", "nu", "alpha", "T0", "beta", "gamma", "s_z",
+        "l10meff", "sigLM", "Mmin", "L0", "precision_fftlog",)
```

Comment:
cM last

Reply:
done

Reply:
Doesn't show as done

Reply:
oops indeed, that one was a stray because it lives in a different file, sorry! fixed now